src/lib.rs

```diff
@@ -1058,7 +1058,7 @@ mod benches {
     use super::{Message, Secp256k1};
 
     #[bench]
-    #[cfg(any(feature = "alloc", feature = "std"))]
+    #[cfg(all(feature = "std", feature = "rnd-std"))]
```

Comment:
`rand-std` should imply `std` so just `rand-std` should be sufficient but [currently it's not the case](https://github.com/rust-bitcoin/rust-secp256k1/blob/7a00b8310d2496c40bcd8597880554ff6271f32a/Cargo.toml#L27) - this is a bug. Also you seem to be missing `a`, so if you're going to fix it anyway might as well add `std` to the Cargo.toml list.

Reply:
Lol, this broken change makes the PR title amusing :)